ledger/metrics/src/main/scala/com/daml/telemetry/Telemetry.scala

```diff
@@ -109,7 +112,7 @@ object DefaultTelemetry extends DefaultTelemetry(OpenTelemetryTracer)
   *
   * It always returns NoOpTelemetryContext, and just executes without modification any given code block function.
   */
-object NoOpTelemetry extends Telemetry(Tracer.getDefault) {
+object NoOpTelemetry extends Telemetry(NoOpTracer) {
```

Comment:
Was the defaul a noop and now it isn't anymore?

Reply:
Yup, it is `package-private` now.